prisma/migrations/20220504191324_user_delete_cascade/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- DropForeignKey
+ALTER TABLE "JournalEntry" DROP CONSTRAINT "JournalEntry_authorId_fkey";
+
+-- AddForeignKey
+ALTER TABLE "JournalEntry" ADD CONSTRAINT "JournalEntry_authorId_fkey" FOREIGN KEY ("authorId") REFERENCES "User"("id") ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
Allows for the user to be deleted if they have journal entries, deletes the journal entries as well